src/server/server.cc

```diff
@@ -63,6 +64,7 @@ Server::Server(engine::Storage *storage, Config *config)
 
   // init cursor_dict_
   cursor_dict_ = std::make_unique<CursorDictType>();
+  crc64_init();
```

Comment:
Since it is not related to Server, I think it can be put into the main function.